README.md

```diff
@@ -11,6 +11,8 @@ The backend Node.js web-service and smart contracts are located in separate repo
 
 `@klimadao/carbonmark-api` -> A [Fastify](https://www.fastify.io/) API acting as a Backend-for-Frontend for Carbonmark
 
+`@klimadao/carbonmark-data` -> data.klimadao.finance site
```

Comment:
I just updated the hostname in commit [e516b45](https://github.com/KlimaDAO/klimadao/pull/1471/commits/e516b45c090863301476b9f1892724c6c0815c7a)
I created #1481 to change the directory name. @sprrwhwk we will need to synchronize to avoid ugly merge conflicts when doing this one.